index.html

```diff
@@ -115,6 +115,41 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Replace the text `*TEMPLATE:` etc. with your name, like what Tuhin did in line 153. This will make it more clear for future contributors to identify the template.

Reply:
I will change it right now!